openmdao_book/features/core_features/running_your_models/set_get.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>I think we really want to break this into a bunch of smaller cells so that the reader can see the prints as they are executed, rather than having them all dumped at the end. Compare directly to the docs page at: http://openmdao.org/twodocs/versions/latest/features/core_features/running/set_get.html</p><p> </p><p>This also applies to other feature docs you've done and going forward, as it was an intentional choice by the feature doc writer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Changed